### ISSUE TITLE
Need a static curl binary package (i686 may have a broken /usr/bin/curl)

### ISSUE THREAD

**Issue by satmandu** [skycocker/chromebrew#4493]
Playing with [dockerized containers](https://gist.github.com/satmandu/791dd6e982d365dda6bbba5c332f0f73), it appears that at least zgb on i686 may have a broken /usr/bin/curl.

Installing a static ```/usr/local/bin/curl_static``` would solve this issue.

My script currently uses a variant on this docker script to build a static curl: https://github.com/dtschan/curl-static/raw/master/build.sh

**Comment by uberhacker** [comment]
These 2 lines break every build on i686:
```
    ENV['CFLAGS'] = '-fuse-ld=lld'
    ENV['CXXFLAGS'] = '-fuse-ld=lld'
```
The curl installed from core does solve this problem.  On i686, the change to `/usr/bin/curl` in the crew script killed curl.

**Comment by satmandu** [comment]
What is your machine model?

I want to build a container with your machine's recovery and see what
happens. (Currently testing i686 with zgb).

**Comment by uberhacker** [comment]
```
$ cat /etc/lsb-release 
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={C776D42E-287A-435E-8BA7-E770BD30B46D}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={C776D42E-287A-435E-8BA7-E770BD30B46D}
CHROMEOS_RELEASE_BOARD=x86-alex_he-signed-mp-v4keys
CHROMEOS_RELEASE_BRANCH_NUMBER=72
CHROMEOS_RELEASE_BUILDER_PATH=x86-alex_he-release/R58-9334.72.0
CHROMEOS_RELEASE_BUILD_NUMBER=9334
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=58
CHROMEOS_RELEASE_DESCRIPTION=9334.72.0 (Official Build) stable-channel x86-alex_he 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=9334.72.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=9334.72.0
```
I'd really like to put the breaks on updates until this is fixed.  We're trying to change way too much all at once and it makes the project difficult to maintain.

**Comment by uberhacker** [comment]
I think the big thing we need to focus on for i686 is to get all packages to build WITHOUT needing any alternate `LD_LIBRARY_PATH` settings and other weird config.  EVERYTHING should work with just `export LD_LIBRARY_PATH=/usr/local/lib`.  This is what is set out of the box.

**Comment by uberhacker** [comment]
I had to do some digging but the recovery image is [here](https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_9334.72.0_x86-alex-he_recovery_stable-channel_alex-mp-v4.bin.zip).

**Comment by satmandu** [comment]
Rust just installed fine in a package based upon that recovery image...

**Comment by uberhacker** [comment]
Fixed in #4532.
